Application/EdFi.Ods.WebApi/Program.cs

```diff
@@ -57,18 +57,18 @@ static void ConfigureLogging(HostBuilderContext hostBuilderContext, ILoggingBuil
 
             async Task ConfigureHostUsingPlugins()
             {
-                var pluginTypes = TypeHelper.GetPluginTypes();
+                var hostConfigurationActivities = TypeHelper.GetAssemblyTypes<IHostConfigurationActivity>();
 
-                foreach (Type pluginType in pluginTypes)
+                foreach (Type hostConfigurationActivity in hostConfigurationActivities)
                 {
                     try
                     {
-                        var plugin = (IPlugin) Activator.CreateInstance(pluginType);
-                        plugin.ConfigureHost(hostBuilder);
+                        var plugin = (IHostConfigurationActivity) Activator.CreateInstance(hostConfigurationActivity);
+                        plugin!.ConfigureHost(hostBuilder);
                     }
                     catch (Exception ex)
                     {
-                        await Console.Error.WriteLineAsync($"Error occured during host configuration by plugin '{pluginType.Name}': {ex}");
+                        await Console.Error.WriteLineAsync($"Error occured during host configuration by plugin '{hostConfigurationActivity.Name}': {ex}");
```

Comment:
We should update the message to use the term "host configuration activity" rather than "plugin":
```csharp
await Console.Error.WriteLineAsync($"Error occurred during host configuration activity '{hostConfigurationActivity.Name}': {ex}");
```